### DISCUSSION TITLE
Moddable <Engine/> particle emitters

### DISCUSSION THREAD

**Discussion by Ek-Videogames** [FakeFishGames/Barotrauma#7543] [Ideas and suggestions]
The particle system used by the <Engine/> component is hardcoded to use bubbles. A nice modding feature would be customization like that of other machines such as turrets, to allow new content mods or overwrite mods to make changes - or remove the particle emitted entirely for visual or performance motives.

**Comment by MateusJK15** [comment]
And it doesn't even makes sense, the bubbles coming from the engines are generated by a phenomenon called cavitation, but the deeper a submarine goes, the harder is for cavitation to occur because of the water pressure, at a depth of 500m even a very fast submarine (like the Alfa class, reaching 41 knots or 76 km/h) wouldn't cavitate, and in this game the player never goes shallower than 1000m.
